src/commands/test.rs

```diff
@@ -345,6 +366,35 @@ pub async fn run_submission_tests(
                 "Score".if_supports_color(OwoStream::Stdout, |text| { text.bold() }),
                 score
             ));
+            let path = global.project.join("README.md");
+            with_locked_file(
+                |file| {
+                    let score = score.clone();
+                    let url = global.aqora_url().unwrap();
+                    async move {
+                        let mut contents = String::new();
+                        file.read_to_string(&mut contents).await?;
+                        file.seek(SeekFrom::Start(0)).await?;
+                        file.write_all(
```

Comment:
I'm not sure I understand how this would replace an existing badge? I think you'd have to wrap it in some comments or parse the markdown to find the existing badge

Reply:
No! I want to inject a new badge each time with the latest score the user achieved! So basically, we have two types of badges:
1. A default use-case badge, which redirects to the Aqora use case.
2. A score badge that displays the user’s score!

Reply:
Maybe it’s not ideal to keep all badges? I thought they could display either progression or regression, but users can always edit the markdown themselves at the end if they want to remove some of them!

Reply:
I would say we should only have one badge generated at a time. thoughts @stubbi ?

Reply:
Oh no, I didn't express myself clearly. There is only one generated badge. However, in the markdown before any test, there is already a badge that refers to Aqora. Then, when you run a test, a new badge with a score is generated, and each test adds a new score badge!